src/lib/payments/accounts.ts

```
@@ -42,7 +42,7 @@ export async function StripeCreateAccount() {
 export async function StripeCreateAccountLink(stripe_account: string) {
     return await stripe.accountLinks.create({
         account: stripe_account,
-        refresh_url: env.BASE_URL + `/api/stripe/${stripe_account}/reauth`,
+        refresh_url: env.BASE_URL + `/api/stripe/reauth`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)